backend/src/components/facilities.js

```diff
@@ -28,7 +28,7 @@ async function getFacilityContacts(req, res) {
 
 async function getFacilityLicences(req, res) {
   try {
-    const operation = `ofm_licences?$select=ofm_health_authority,ofm_licence,ofm_licenceid,statuscode&$filter=(_ofm_facility_value eq ${req.params.facilityId})`
+    const operation = `ofm_licences?$select=ofm_health_authority,ofm_licence,ofm_licenceid,statuscode&$filter=(_ofm_facility_value eq ${req.params.facilityId}) and (statuscode eq 1)`
```

Comment:
I am not sure if in the future there will be more status codes for this one and they want to display them as well? I normally use statecode if I want to decide an entity is Active or Inactive.

Reply:
Found instances of existing use of statecode on filters, will push through for sake of sprint and ask Hoang's confirmation if these need to change to statecode.